course5/homework6/string.cc

```diff
@@ -0,0 +1,115 @@
+#include <memory>
+#include <cstring>
+
+#include "string.h"
+
+namespace jds {
+
+String::String()
+{
+	mLength = 0;
```

Comment:
Just set the default value in the class definition, and get away with compiler-generated constructor.